L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

```diff
@@ -974,14 +997,59 @@ void l1t::TriggerMenuParser::parseDeltaPhi_Cos_LUTS(std::map<std::string, tmeven
     lutName += "-";
     lutName += obj2;
     m_gtScales.setLUT_DeltaPhi(lutName,lut_delta_phi,prec1);
-        
+
     // Second Get the Cosh for this delta phi Set
     std::vector<long long> lut_cos;
     applyCos(val_delta_phi, n);
     setLut(lut_cos, val_delta_phi, prec2);
     m_gtScales.setLUT_Cos(lutName,lut_cos,prec2);
 
-}	
+}
+
+void l1t::TriggerMenuParser::parsePhi_Trig_LUTS(std::map<std::string, tmeventsetup::esScale> scaleMap, std::string obj, std::string func, unsigned int prec)
```

Comment:
done.

yes, only two valid values "sin" and "cos", now passed as const &. 
Will request developer to use enums in the next version.  I hope OK for now.